docker-compose.balena.yml

```diff
@@ -1,5 +1,19 @@
 version: "2"
 services:
+  srly-ose-wifi-connect:
+    image: screenly/srly-ose-wifi-connect:BOARD_TAG
+    build: ./wifi-connect # @TODO: Remove this.
```

Comment:
Remove the `build` option since we'll be fetching the images from Docker Hub instead.

Reply:
Remove the `:BOARD_TAG` part as well.